configure.ac

```diff
@@ -761,6 +761,9 @@ define(MINIMUM_REQUIRED_BOOST, 1.47.0)
 
 dnl Check for boost libs
 AX_BOOST_BASE([MINIMUM_REQUIRED_BOOST])
+if test x$want_boost = xno; then
```

Comment:
Shouldn't this check be right after use_boost, not after AX_BOOST_BASE? Otherwise it will still do a boost version check even though no boost was requested.

Reply:
That's what I tried at first too, but it seems to spawn `checking whether the Boost::System library is available... yes` Any ideas why?

Reply:
No, no idea, that's kind of where my autotools knowledge stops. Maybe @theuni can help.